editor/code/views/info/media/index.css

```diff
@@ -4,3 +4,7 @@ body {
   font-family: var(--vscode-editor-font-family);
   /* font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif; */
 }
+
+#root {
+  height: 100vh;
```

Comment:
Had to go with this to get the `overflow` working, don't know if it's the best approach (using 100% wouldn't work).

Reply:
I see, can we add a test case in examples that overflows both the error panel and the goal panel?

This way we can tweak that.